install/Knative-with-ICP.md

```diff
@@ -28,14 +28,14 @@ curl -L https://github.com/knative/serving/releases/download/v0.2.2/release-lite
 If the `image-security-enforcement` enabled when you install [IBM Cloud Private](https://www.ibm.com/cloud/private). You need to update the [image security policy](https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.1/manage_images/image_security.html) allow to pull the knative image.
 Using the following commend get the image security policy.
 ```
-# kubectl get clusterimagepolicies
+kubectl get clusterimagepolicies
```

Comment:
I'd like you add a `$` before those command

Reply:
We would like to keep it consistent across our docs, so please don't add a `$`

Reply:
+1 to what Averi said.

Reply:
OK, let's keep consistent, thanks @samodell and @averikitsch